### PR TITLE
SOC reader, simplisma, fix

### PR DESCRIPTION
**Pull request by atravert**
Add Surface optics reader
- [x] related to #455 
- [x] Tests have been added (mostly required)
- [x] User-visible changes  documented in `CHANGELOG.md` 
- [x] The new methods have been listed in `docs/userguide/reference/api.rst`.
- [ ] Sample files and Examples to be added after authorization of file owner  !

Deprecate verbose option in simplisma for consistency with mcarls
- [x] User-visible changes  documented in `CHANGELOG.md` 

Put back `__init__.py` in test_readers (caused an error on my fork for tests with py 3.9)

### CODE REVIEW

File: spectrochempy/core/readers/importer.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 FILETYPES = [
     ("scp", "SpectroChemPy files (*.scp)"),
     ("omnic", "Nicolet OMNIC files and series (*.spa *.spg *.srs)"),
+    ("soc", "Surface Optics Corp. (*.ddr *.hdr *.srs)"),
```
Comment (fernandezc) [inline spectrochempy/core/readers/importer.py:29]: *.sdr instead of *sds

File: tests/test_core/test_readers/test_read_soc.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+import spectrochempy as scp
+
+
+def test_read_SOC():
+    """upload and read surface oftics exemple"""
+
+    # the following does not work
+    baseurl = "https://github.com/chet-j-ski/SOC100_example_data/raw/main/"
```
Comment (fernandezc) [inline tests/test_core/test_readers/test_read_soc.py:13]: I am not sure If I have tested this, but I think this is not yet possible in read_remote. Anyway the best is to add the example files in spectrochempy_data.

**Review by fernandezc — APPROVED** [review]

**Comment by fernandezc** [comment]
> Put back `__init__.py` in test_readers (caused an error on my fork for tests with py 3.9)

Regarding this I do not understand. It works for me. And the tests seems to pass on GitHub.

File: tests/test_core/test_readers/test_read_soc.py (hunk 2 above)
Comment (atravert) [reply tests/test_core/test_readers/test_read_soc.py:13]: Yes, but I will ask the file owner first, and then add them in a next PR.

File: spectrochempy/core/readers/importer.py (hunk 1 above)
Comment (atravert) [reply spectrochempy/core/readers/importer.py]: done.

**Pull request merged**